helmfile/values/kube-prometheus-stack-sc.yaml.gotmpl

```diff
@@ -188,7 +188,7 @@ kubeScheduler:
 
 grafana:
   image:
-    tag: 9.3.13
```

Comment:
Should this not be upgrade when we do kube-prometheus-stack? Same as before, why do we have this set.
Both will follow the charts app version.

Reply:
I upgraded both based on a previous [PR](https://github.com/elastisys/compliantkubernetes-apps/pull/1284/files), as for why we specify the tag here as well, I do not have a good answer :sweat_smile:

Reply:
@OlleLarsson @crssnd you maybe have some input about these two variables?

Reply:
We seem to perhaps have overriden the defaults for some time, but at least it looks like we chose a newer [patch version for security fixes](https://github.com/elastisys/compliantkubernetes-apps/commit/611e4f454c4987b50ecb7188224f10887c1b58fa).

Anyhow, we should remove and use the default upstream if possible.

Reply:
we should also update the [grafana chart](https://github.com/elastisys/compliantkubernetes-apps/blob/main/helmfile/upstream/kube-prometheus-stack/Chart.yaml#L23) used by kube-prometheus-stack

Reply:
I would say no, then we'll modify an upstream chart which is not something that we want to do.
We should instead upgrade the kube-prometheus-stack chart to a version that has the correct dependencies :)

Reply:
It may be to complicated to upgrade the entire kube-prometheus-stack only to upgrade the Grafana chart, especially because we have some manual changes done to the upstream dashboards.
For me it's good enough to have only the Grafana chart dependency updated for now.

Reply:
We have this issue https://github.com/elastisys/compliantkubernetes-apps/issues/1293